src/transformStateWithClones.js

```diff
@@ -8,34 +8,36 @@
  */
 
 function transformStateWithClones(state, actions) {
-  const resultArr = [];
-  const createNewObj = (obj) => {
-    const newObj = resultArr.length === 0
-      ? Object.assign({ ...state })
-      : Object.assign({ ...resultArr[resultArr.length - 1] });
+  const results = [];
```

Comment:
This is from the checklist of one previous task. 

> 3. [NAMING] - use proper names for variables - if you want to name variable result, pick a better one :)

let's give more descriptive name for this variable.

Here are some good examples:
`stateSnapshots`
`stateHistory`